pytube/extract.py

```diff
@@ -98,6 +98,25 @@ def is_age_restricted(watch_html: str) -> bool:
     return True
 
 
+def playability_status(watch_html: str) -> (str, str):
+    """Return the playability status and reason of a video.
```

Comment:
Return the playability status and rationale given by Youtube for that status  of a given video. For example a video can be `RecordingUnavailable` with the explanation that `This live stream recording is not available`.

Reply:
I'll update this in a moment